app/src/main/java/gateway/controller/CtrlAuthLogin.java

```diff
@@ -48,19 +50,24 @@ public static ResSession auth_login (Credentials credentials)
 			if (statusCode == 201) {
 				// If the status code is 201, indicating login succeed, initialize an Authorization
 				// object in the response and extract the JWT token.
+				res.code = 200;
```

Comment:
Don't set the return code yourself but take it from the authentication service response. 

Like `res.code = response.statusCode()`, this could replace line 47.